Packs/McAfee-TIE/Integrations/McAfee-TIE_V2/McAfee-TIE_V2.py

```diff
@@ -0,0 +1,603 @@
+        FileEnterpriseAttrib.AVG_LOCAL_REP: "Avg_Local_Rep",
+        FileEnterpriseAttrib.PARENT_MIN_LOCAL_REP: "Parent_Min_Local_Rep",
+        FileEnterpriseAttrib.PARENT_MAX_LOCAL_REP: "Parent_Max_Local_Rep",
+        FileEnterpriseAttrib.PARENT_AVG_LOCAL_REP: "Parent_Avg_Local_Rep",
+        FileEnterpriseAttrib.FILE_NAME_COUNT: "File_Name_Count",
+        FileEnterpriseAttrib.DETECTION_COUNT: "Detection_Count",
+        FileEnterpriseAttrib.LAST_DETECTION_TIME: "Last_Detection_Time",
+        FileEnterpriseAttrib.IS_PREVALENT: "Is_Prevalent",
+        FileEnterpriseAttrib.CHILD_FILE_REPS: "Child_File_Reps",
+        FileEnterpriseAttrib.PARENT_FILE_REPS: "Parent_File_Reps",
+    }
+
+    def parse_attributes(self, attributes: Dict[str, Any]) -> Dict[str, Any]:
+        parsed_res: Dict[str, Any] = {}
+        for key, val in attributes.items():
+            if(key == FileEnterpriseAttrib.FIRST_CONTACT or key == FileEnterpriseAttrib.LAST_DETECTION_TIME):
+                parsed_res[self.ATTRIBUTES_KEYS[key]] = epoch_to_localtime(int(val))
+
+            elif(key == FileEnterpriseAttrib.SERVER_VERSION):
+                parsed_res[self.ATTRIBUTES_KEYS[key]] = EnterpriseAttrib.to_version_string(val)
+
+            elif(key == FileEnterpriseAttrib.CHILD_FILE_REPS or key == FileEnterpriseAttrib.PARENT_FILE_REPS):
+
+                parsed_res[self.ATTRIBUTES_KEYS[key]] = val
+            elif(key in self.ATTRIBUTES_KEYS):
+                parsed_res[self.ATTRIBUTES_KEYS[key]] = val
+
+            else:
+                parsed_res[key] = val
+
+        return parsed_res
+
+
+class AtdFileReputationParser(GeneralFileReputationParser):
+    ATTRIBUTES_KEYS = {
+        AtdAttrib.GAM_SCORE: "GAM_Score",
+        AtdAttrib.AV_ENGINE_SCORE: "AV_Engine_Score",
+        AtdAttrib.SANDBOX_SCORE: "Sandbox_Score",
+        AtdAttrib.VERDICT: "Verdict",
+        AtdAttrib.BEHAVIORS: "Behaviors",
+    }
+    ATD_TRUST_LEVELS = {
+        AtdTrustLevel.KNOWN_TRUSTED: "KNOWN_TRUSTED",
+        AtdTrustLevel.MOST_LIKELY_TRUSTED: "MOST_LIKELY_TRUSTED",
+        AtdTrustLevel.MIGHT_BE_TRUSTED: "MIGHT_BE_TRUSTED",
+        AtdTrustLevel.UNKNOWN: "UNKNOWN",
+        AtdTrustLevel.MIGHT_BE_MALICIOUS: "MIGHT_BE_MALICIOUS",
+        AtdTrustLevel.MOST_LIKELY_MALICIOUS: "MOST_LIKELY_MALICIOUS",
+        AtdTrustLevel.KNOWN_MALICIOUS: "KNOWN_MALICIOUS",
+        AtdTrustLevel.NOT_SET: "NOT_SET",
+    }
+
+    def parse_attributes(self, attributes: Dict[str, Any]) -> Dict[str, Any]:
+        parsed_res: Dict[str, Any] = {}
+        for key, val in attributes.items():
+            if(key in self.ATTRIBUTES_KEYS):
+                parsed_res[self.ATTRIBUTES_KEYS[key]] = val
+
+            else:
+                parsed_res[key] = val
+
+        return parsed_res
+
+
+class UnknownReputationHandler(GeneralFileReputationParser):
+
+    def parse_attributes(self, attributes: Dict[str, Any]) -> Dict[str, Any]:
+        return attributes
+
+
+def validate_certificates_format():
```

Comment:
where is this function used?

Reply:
I deleted this function, it was used to check in v1 if the given certificates are valid, I used a different approach to check.